tests/command/commands/user_test.py

```diff
@@ -194,26 +194,6 @@ def test_handle_edit_github(self):
         self.mock_facade.store.assert_called_once_with(user)
         self.mock_github.org_add_member.assert_called_once_with("rob")
 
-    def test_handle_edit_github_error(self):
```

Comment:
Why was this removed?

Reply:
Because it is a duplicate function (overridden immediately by the one directly below it). It seems to be a remnant of some merge conflict (woooo spooky).

Reply:
Oh haha didn't notice that, nice catch.